### DISCUSSION TITLE
Feature Request: Interaction with the Standard Interface

### DISCUSSION THREAD

**Discussion by a268-Beefy** [SFX-WoW/Masque#216] [Issues] [answered]
### Game Flavor

Retail

### Game Version
9.0.2.36753

### Add-On Version
9.0.4

### Description

Request to add feature to apply appearence to standart Blizzard buff/debuffs icons.

**Comment by StormFX** [comment, marked as answer]
Masque, at its core, is a library. This means that add-ons (including the default UI) must support Masque, not the other way around. For default Buff Frame support, try [here](https://www.curseforge.com/wow/addons/blizzbuffsfacade).
